test/controllers/uas.spec.js

```diff
@@ -0,0 +1,1033 @@
+/* global describe, it, xit, sinon, expect */
+// var sinon = require("sinon");
+var should = require("chai").should();
+var expect = require("chai").expect;
+
+var UAS = require("../../src_new/controllers/uas.js");
+var UTIL = require("../../src_new/util.js");
+var AM = require("../../src_new/adapterManager.js");
+var CONSTANTS = require("../../src_new/constants.js");
+var CONFIG = require("../../src_new/config.js");
+var BM = require("../../src_new/bidManager.js");
+var SLOT = require("../../src_new/slot.js");
+
+var commonDivID = "DIV_1";
+
+describe("CONTROLLER: UAS", function() {
```

Comment:
At most of the cases only "a function is called" is checked

Reply:
Yes I am not completed with test cases